### PR TITLE
Konyang: Air Konyang Vessel

### PR DESCRIPTION
**Pull request by RustingWithYou**
Adds a new away site to the Haneunim sector, an Air Konyang passenger vessel with ghostroles for the pilot, crew and passengers.

### CODE REVIEW

**Comment by RustingWithYou** [comment]
!review

File: maps/away/ships/konyang/air_konyang/air_konyang.dm [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+	shiptype = "Civilian passenger transport."
+	max_speed = 1/(2 SECONDS)
+	burn_delay = 1 SECONDS
+	vessel_mass = 5000
+	fore_dir = SOUTH
+	vessel_size = SHIP_SIZE_SMALL
+	shuttle = "Air Konyang Transport"
+
+/obj/effect/overmap/visitable/ship/landable/air_konyang/New()
+	designation = "[pick("John Ship, CEO of Ships")]"
```
Comment (DreamySkrell) [inline maps/away/ships/konyang/air_konyang/air_konyang.dm:43]: .

**Comment by DreamySkrell** [comment]
image

Grates should be used to hide pipes/wires/vents/etc (but still show they are there) in like the "public" areas like hallways or department rooms, or in this case the passenger cabin.

Avoid overusing grates like this in like the "engineering"/"maintenance" areas, cause they make it harder to see how pipes and wires connect and are laid out, and make it annoying to repair or rearrange them (ofc in this case that probably won't be done, but it still good to establish and follow some standards).

**Comment by DreamySkrell** [comment]
image
I don't think this is good (lights under grates) - it makes it hard to follow the pipes and wires.

**Comment by DreamySkrell** [comment]
image
Make that airlock three tiles wide like this ^ so it is easier to leave/enter the ship when docked.

**Comment by DreamySkrell** [comment]
image
Windows on engines is a bit of a old mapping style, and personally I do not like it. It's... weird to see the engine working from inside the ship/shuttle.

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
see previous comments

**Comment by DreamySkrell** [comment]
sync/merge from master to get fixed mapdiffbot preview

**Review by DreamySkrell — APPROVED** [review]
image
broken flag

otherwise looks fine

**Review by NonQueueingMatt — APPROVED** [review]
